Pod/Classes/MockMatcher.swift

```diff
@@ -25,7 +25,7 @@ public class MockMatcher {
             result = false
         }
         
-        for var index=0; index<firstArray.count && result; index++ {
+        for var index=0; index<firstArray.count && result; index += 1 {
```

Comment:
This C-style `for` syntax will be deprecated too in Swift 3. Let's use a `for in` syntax:
```swift
for index in 0..<firstArray.count {
    result = match(firstArray[index], secondArray[index])
}
```